.gitignore

```diff
@@ -24,6 +24,6 @@ _testmain.go
 *.prof
 bin
 pkg
-src/*
-gobu
+#src/*
```

Comment:
otherwise i couldnt check in changes in the source file. i will prove if that it is needed.